pom.xml

```diff
@@ -89,7 +89,7 @@
       <dependency>
         <groupId>org.robolectric</groupId>
         <artifactId>android-all</artifactId>
-        <version>5.0.0_r2-robolectric-0</version>
+        <version>5.0.0_r2-robolectric-1</version>
```

Comment:
The change is in install-dependencies.sh, all classes defined in layoutlib.jar are used to update/override classes in android-all.jar.